.github/workflows/codeql.yml

```diff
@@ -33,6 +33,8 @@ jobs:
 
       - name: Setup Environment
         uses: ./.github/actions/setup
+        with:
+          gh-token: ${{ secrets.GITHUB_TOKEN }} # secrets must be passed to composite actions
```

Comment:
AFAICT, we can `skip-compact` here bc codeql isn't analyzing the artifacts, just the js/ts source code. Do you agree?

Reply:
> AFAICT, we can skip-compact here bc codeql isn't analyzing the artifacts, just the js/ts source code. Do you agree?

Yes, also CodeQL doesn't support analysis of `*.cjs` files produced by the compact compiler anyway ([src](https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/#languages-and-compilers))